README.md

```diff
@@ -10,13 +10,13 @@ You can install the development version from GitHub, via `devtools::install_gith
 
 ### Kusto Endpoint Interface
 
-Connect to a Kusto cluster by instantiating a `kusto_query_endpoint` object with the cluster URI, database name, and an `AzureRMR::AzureToken` object, which you can obtain via the `get_kusto_token` helper function.
+Connect to a Kusto cluster by instantiating a `kusto_database_endpoint` object with the cluster URI, database name, and an `AzureRMR::AzureToken` object, which you can obtain via the `get_kusto_token` helper function.
 
 ```r
 
 library(AzureKusto)
 
-Samples <- kusto_query_endpoint(server="https://help.kusto.windows.net",
+Samples <- kusto_database_endpoint(server="https://help.kusto.windows.net",
```

Comment:
This example isn't working anymore. Throws `Error: Invalid/unsupported property name: .azure_token`

Reply:
I renamed the `.azure_token` arg to `.query_token`. This is because we also have an ingestion URI, and it's possible that in the future we might need to have ingestion tokens. Otherwise it should work as before.